src/main/java/frc/robot/commands/AutoBalanceCommandGroup.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
You could create a class for AutoBalanceCommandGroup like you're doing here.  But there might be an easier alternative: Any time you want to say
```
Command autoCmd = new AutoBalanceCommandGroup(drivetrainSub)
```
you could instead just say
```
Command firstCmd = new DriveForwardCommand(drivetrainSub);
Command secondCmd = new DriveUpTheRampCommand(drivetrainSub);
Command autoCmd = firstCmd.andThen(secondCmd);
```
Then you wouldn't need AutoBalanceCommandGroup at all, and you could delete the entire file.